problem_builder/tests/integration/test_clarifications.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2014-2015 Harvard, edX & OpenCraft
+#
+# This software's license gives you freedom; you can copy, convey,
+# propagate, redistribute and/or modify this program under the terms of
+# the GNU Affero General Public License (AGPL) as published by the Free
+# Software Foundation (FSF), either version 3 of the License, or (at your
+# option) any later version of the AGPL published by the FSF.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program in a file in the toplevel directory called
+# "AGPLv3".  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Test that <span class="pb-clarification"> elements are transformed into LMS-like tooltips.
+"""
+
+# Imports ###########################################################
+import ddt
+from cgi import escape
+from xblockutils.base_test import SeleniumXBlockTest
+
+
+# Classes ###########################################################
+
+
+@ddt.ddt
+class ClarificationTest(SeleniumXBlockTest):
+    """
+    Test that the content of span.pb-clarification elements is transformed into
+    tooltip elements.
+    """
+
+    clarification_text = 'Let me clarify...'
+
+    mcq_template = """
+        <problem-builder>
+            <pb-mcq question="Who was your favorite character? {clarify}">
+                <pb-choice value="gaius">Gaius Baltar</pb-choice>
+                <pb-choice value="adama">Admiral William Adama {clarify}</pb-choice>
+                <pb-choice value="starbuck">Starbuck</pb-choice>
+            </pb-mcq>
+        </problem-builder>
+    """
```

Comment:
If you want, you can put these templates into the `xml_templates` folder, and change the placeholders to `{{django_template syntax}}`, as some of the other tests do. But this way is fine too; personally I'd probably leave it as-is.